lib/upstream/core.js

```diff
@@ -91,6 +91,19 @@ class Core {
     });
   }
 
+  destroy(fourfour, onComplete) {
+    return this._url((err, url) => {
+      if (err) return onComplete(err);
+      request.del({
+          url: `${url}/views/${fourfour}`,
+          headers: this._headers(),
+          json: true
+        })
+        .on('response', this._onResponseStart(onComplete))
```

Comment:
in the case that a response is never returned, ie: the client times out before the service sends a response code, an error is emitted, because we never get a response.

in the case of a 4xx or 5xx, a response event is emitted, and then we need to check if it's actually 2xx